README.md

```diff
@@ -62,3 +62,13 @@ holepunch foo_bar --cidr=192.168.0.0/16 22 80
 # Also works with IPv6 ranges
 holepunch foo_bar --cidr=2001:882f::1/128 443
 ```
+
+Less verbose or quiet mode:
+
+```
+# Verbose mode
+holepunch foo_bar -v2 --cidr=192.168.0.0/16 22 80
+
+# Quiet mode
+holepunch foo_bar -v0 --cidr=2001:882f::1/128 443
+```
```

Comment:
Overall I like this idea, but let's split it out into its own PR so we can address it separately from the bug fix.

Reply:
Sure. I'll create another PR for verbose mode

Reply:
@erik created PR for verbose mode. Let me know what else I need to update. PR https://github.com/erik/holepunch/pull/19